notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -1,1746 +1,1461 @@
 {
```

Comment:
<p>For consistency use class_labels variable instead of class names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Ok, done.</p>